### DISCUSSION TITLE
Forwarding MQTT traffic

### DISCUSSION THREAD

**Discussion by lordratner** [hassio-addons/addon-nginx-proxy-manager#276] [Q&A]
I’m running HAOS on an RPi4 and using NGINX Proxy Manager (0.11.0) to use https://ha.example.com to access HA with SSL. Works great.

I have an off-site OpenSprinkler that I want to use MQTT with. I have the Mosquitto broker addon installed. With everything on my LAN using 192.168.xx.xx:1883 it works great. But when I move to offsite and try to use mqtt.example.com as the broker address, nothing works.

I have NGINX forwarding mqtt.example.com to 192.168.xx.xx:1883, but no luck. I’ve tried with the Sprinkler using a broker address of mqtt.example.com on both port 80 and 1883.

Any idea what I’m doing wrong? Can NGINX do this?

Thanks,
Seth

**Comment by frenck** [comment]
You need to use port forwarding in your router. The Nginx Proxy Manager is an HTTP proxy, not a router.

../Frenck
